Environment/AeroAllergenObserved/README.md

```diff
@@ -0,0 +1,42 @@
+# Aero Allergen Observed
+
+## Description
+This entity models aero allergens observed at a given location and
+  related overall allergen risk.
+
+**Note**: JSON Schemas only capture the NGSI simplified representation, this
+  means that to test the JSON schema examples with
+  a [FIWARE NGSI version 2](http://fiware.github.io/specifications/ngsiv2/stable)
+  API implementation, you need to use the `keyValues`
+  mode (`options=keyValues`).
+
+## Examples of Use
+
+```json
+{
+  "id": "AeroAllergenObserved-CDMX-Pollen-Cuajimalpa",
+  "type": "AeroAllergenObserved",
+  "alnusLevel": "moderate",
+  "alnusConcentration": 40,
+  "alnusAllergenicity": "3",
```

Comment:
why is this a number?  is it needed to be between quotes?

Reply:
As by spec: As this can be different between countries, regulations or implementations, the set of allowed values will depend on the reference specification used.